queue_services/account-mailer/src/account_mailer/resources/worker.py

```diff
@@ -437,6 +446,7 @@ def handle_other_messages(message_type, email_msg):
         QueueMessageTypes.PAD_SETUP_FAILED.value,
         QueueMessageTypes.PAYMENT_PENDING.value,
         QueueMessageTypes.EJV_FAILED.value,
```

Comment:
Shouldn't EJV_FAILED and CSV_FAILED just be `QueueMessageTypes.PAYMENT_RECONCILIATION_FAILURE`